### DISCUSSION TITLE
In the table columns can we add the author/creator of the issue

### DISCUSSION THREAD

**Discussion by abhinavbhagwat** [community/community#14031] [Projects and Issues]
Currently when we list the issues in the project we can add various columns. One important column to have will be the author who created the issue. Current we have to open the issue to know who created the issue. Will be good to have it in the tabluar listing.

<img width="1789" alt="image" src="https://user-images.githubusercontent.com/11055594/161240249-c474bb8f-a191-4bae-be33-240e38316fe7.png">

Add Author/Creator of the issue in the above columns.

**Comment by adrian-ionescu** [comment]
Yes, please! I was trying to build an overview of my team's PRs and I was surprised to see we can display e.g. Assignees (which is something optional and often unused), whereas PR Author (something implicit) is not an option. Also noticed that I can't create a custom column titled "Author" - it says that the name is reserved. So hopefully it's trivial to just expose that.

**Comment by usbtc-jeff** [comment]
I would also appreciate this addition. It's a useful way to sort through issues. It would help me find the issues I created to make sure they are assigned and progressing.

**Comment by rvashishth** [comment]
This is now a must have feature for our teams. It's causing too much manual efforts to track/manage issue authors.

**Comment by agilesteel** [comment]
Any updates on this? It's been almost 2 years since this issue was raised.

**Comment by Grisgram** [comment]
Yes, this is very important for every project that is not a "one-man-show" -- author please!

**Comment by richardliaw** [comment]
This would be awesome to have!

**Comment by felixphool** [comment]
If you're looking to display the author/creator of an issue in a table, you can do this using GitHub’s API or markdown tables (if manually updating).

Manual Approach (Markdown Table)
If you’re creating a table manually, just add an "Author" column:
| Issue # | Title              | Author  | Status  |
|---------|--------------------|---------|---------|
| #101    | Fix login bug      | @user1  | Open    |
| #102    | Add dark mode      | @user2  | Closed  |

API Approach (For Automation)
If you're fetching issues dynamically, GitHub’s REST API provides a user.login field under issue. Example using cURL:
curl -s "https://api.github.com/repos/OWNER/REPO/issues" | jq '.[] | {number, title, user: .user.login, state}'
This will return issue numbers, titles, authors, and statuses. You can then format it into a table.

Hope this helps. Let me know if you have a better way.

**Reply by richardliaw** [reply]
i think this is unrelated; I believe the original topic is about having Author in a Github Project, not in an issue.

**Comment by yongruilin** [comment]
This would be very helpful feature!
